src/cbor.c

```diff
@@ -571,11 +571,14 @@ cbor_encode_largeblob_key_ext(cbor_item_t *map)
 }
 
 cbor_item_t *
-cbor_encode_cred_ext(const fido_cred_ext_t *ext)
+cbor_encode_cred_ext(const fido_cred_t *cred)
```

Comment:
should we consider passing `cred->ext` and `cred->blob` here to narrow down the scope of `cred` attributes that the function has access to?

Reply:
fixed in f3271ce7be5c58e8263da49a710f8ac483cdc819